acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -324,4 +330,49 @@ public void allOtherDetailsAreHidden() throws Throwable {
             consumablePublicationPage.getDisplayedElementsThatShouldBeHiddenForUpcomingPublication(), is(empty())
         );
     }
+
+    @Given("^I have a released publication with nominal date falling before (\\d+) weeks from now$")
+    public void iHaveAReleasedPublicationWithNominalDateFallingBeforeWeeksFromNow(final int weeksFromNow)
+        throws Throwable {
+
+        final Publication publicationWithNominalDateBeforeCutOff = TestDataFactory.createPublicationWithNoAttachments()
+            .withNominalDate(Instant.now()
```

Comment:
The alternative is we don't use real time and use a mockable time provider to avoid the tests changing depending on which day you run them on. Maybe worth considering?

Reply:
Using custom, mockable time provider is something I typically do but, as discussed, this is a (largely black-box style) acceptance test where the application is started in a standalone mode and where swapping time provider implementation on the fly, while definitely doable, is relatively cumbersome (and debatable whether it it should be done in this context). At the same time, dealing with time by manipulating the Nominal Publication Date value was quite easy and worked well enough (including being sufficiently stable and predictable which is often a problem in time-involving tests) so I settled on it rather than introduce a more elaborate mechanism that is not absolutely necessary at this point.

Reply:
Agreed. Leave as is.